frontend/src/static/js/v_zoom.js

```diff
@@ -23,6 +37,36 @@ window.vZoom = {
     getSliceLink(slice) {
       return `${window.getBaseLink(this.info.user.repoId)}/commit/${slice.hash}`;
     },
+
+    getNumCommitMessageBody() {
+      let numNonEmptyCommitMessageBody = 0;
+      this.info.user.commits.forEach((commit) => {
+        commit.commitResults.forEach((commitResult) => {
+          if (commitResult.messageBody !== '') {
+            numNonEmptyCommitMessageBody += 1;
+          }
+        });
+      });
+
+      return numNonEmptyCommitMessageBody;
+    },
+
+    toggleAllCommitMessagesBody(isActive) {
+      this.showAllCommitMessageBody = isActive;
+
+      const renameValue = this.showAllCommitMessageBody ? 'body active' : 'body';
+
+      const commitMessageClasses = document.getElementsByClassName('body');
+      Array.from(commitMessageClasses).forEach((commitMessageClass) => {
+        commitMessageClass.className = renameValue;
+      });
+
+      this.numExpandedCommitMessages = isActive ? this.getNumCommitMessageBody() : 0;
+    },
+
+    updateNumExpandedCommitMessages() {
+      this.numExpandedCommitMessages = document.getElementsByClassName('active').length;
```

Comment:
This method seems not safe. If I am not wrong, `document` stands for the whole HTML. You can not guarantee that there are no other `active` class elements in other parts of the HTML.

Reply:
Have modified to get element by class name of `commit-message` instead